### PR TITLE
New R package vcfr

### PR DESCRIPTION
**Pull request by dorton21**
Depends On:
#22493
#22494

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-vcfr/package.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Facilitates easy manipulation of variant call format (VCF) data. Functions
+    are provided to rapidly read from and write to VCF files. Once VCF data is
+    read into R a parser function extracts matrices of data. This information
+    can then be used for quality control or other purposes. Additional
+    functions provide visualization of genomic data. Once processing is
+    complete data may be written to a VCF file (*.vcf.gz). It also may be
+    converted into other popular R objects (e.g., genlight, DNAbin). VcfR
+    provides a link between VCF data and familiar R software."""
+
+    homepage = "https://www.example.com"
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-vcfr/package.py]:
```suggestion
    homepage = "https://github.com/knausb/vcfR"
```

File: var/spack/repos/builtin/packages/r-vcfr/package.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    cran     = "vcfR"
+
+    maintainers = ['dorton21']
+
+    version('1.12.0', sha256='dd87ff010365de363864a44ca49887c0fdad0dd18d0d9c66e44e39c2d4581d52')
+
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-vcfr/package.py:27]: This depends on R (≥ 3.0.1)

File: var/spack/repos/builtin/packages/r-vcfr/package.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    depends_on('r-dplyr', type=('build', 'run'))
+    depends_on('r-magrittr', type=('build', 'run'))
+    depends_on('r-memuse', type=('build', 'run'))
+    depends_on('r-pinfsc50', type=('build', 'run'))
+    depends_on('r-rcpp', type=('build', 'run'))
+    depends_on('r-stringr', type=('build', 'run'))
+    depends_on('r-tibble', type=('build', 'run'))
+    depends_on('r-vegan', type=('build', 'run'))
+    depends_on('r-viridislite', type=('build', 'run'))
+    depends_on('zlib')
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-vcfr/package.py:39]: I do not see this listed as a dependency.

**Review by glennpj — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-vcfr/package.py (hunk 3 above)
Comment (dorton21) [reply var/spack/repos/builtin/packages/r-vcfr/package.py:39]: zlib isn't listed as a dep but without it, the package will not build.

**Review by glennpj — APPROVED** [review]
LGTM

**Pull request merged**